src/platform/arm-nuttx/iotjs_module_gpio-arm-nuttx-stm32f.inl.h

```diff
@@ -0,0 +1,445 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IOTJS_MODULE_GPIO_ARM_NUTTX_STM32F_INL_H
+#define IOTJS_MODULE_GPIO_ARM_NUTTX_STM32F_INL_H
+
+#include "stm32_gpio.h"
+#include "module/iotjs_module_gpio.h"
+
+#define GPIO_FREQUENCY_DEFAULT GPIO_SPEED_25MHz
+#define GPIO_PINCNT_IN_NUTTXPORT 16
+#define GPIO_PORTCNT_IN_NUTTXPORT 2
+
+#define GPIO_CONFIG_OK 0
+
+// Change iotjs pin or port number to nuttx gpio pin or port index
+static inline int32_t GetGpioPortIndexFromPinNumber(int32_t pin) {
```

Comment:
if pin nubmer can be [GPIO_PIN0-GPIO_PIN15], then isn't it more propert to type ``uint32_t`` than ``int32_t``?